app/src/test/java/com/amaze/filemanager/filesystem/files/FileUtilsTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2014-2021 Arpit Khurana <arpitkh96@gmail.com>, Vishal Nehra <vishalmeham2@gmail.com>,
+ * Emmanuel Messulam<emmanuelbendavid@gmail.com>, Raymond Lai <airwave209gt at gmail.com> and Contributors.
+ *
+ * This file is part of Amaze File Manager.
+ *
+ * Amaze File Manager is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.amaze.filemanager.filesystem.files
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class FileUtilsTest {
+
+    /**
+     * Test FileUtils.getPathsInPath()
+     *
+     * @see FileUtils.getPathsInPath
+     */
+    @Test
+    fun testGetPathsInPath() {
+        assertEquals(
+            6,
+            FileUtils.getPathsInPath("/etc/default/grub/2/conf.d").size
+        )
+        assertEquals(
+            7,
+            FileUtils.getPathsInPath("/var/log/nginx/access/2021-01-01/error.log").size
+        )
+        assertEquals(
+            7,
+            FileUtils.getPathsInPath("/system/lib/modules/drivers/net/broadcom/").size
+        )
+        assertEquals(
+            7,
+            FileUtils.getPathsInPath("  /some/nasty/path/with/space/prefixed").size
+        )
+        assertEquals(
+            8,
+            FileUtils.getPathsInPath("/some/nasty/path/with/space  /  in/between").size
```

Comment:
Can add one test with some path which is not starting with a forward slash. I think that's why caused the original crash somehow this method got such path (our own custom paths? like `smb://abcd`?).